### PR TITLE
fix(curriculum): Portuguese challenge tests

### PR DESCRIPTION
**Pull request by huyenltnguyen**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Fixes the failing test caused by PR #36970.

I didn't want to dismiss the author's contribution, so I decided to fix the tests instead of reverting the changes. I fixed them by removing all the assert message arguments, following a Randell's comment in another PR https://github.com/freeCodeCamp/freeCodeCamp/pull/37161#issuecomment-540704059.

The changes were tested locally using the following commands:
```
cd curriculum

npm run test -- -g "Use CSS Selectors to Style Elements"
``` 
With `TEST_CHALLENGES_FOR_LANGS=portuguese`

### CODE REVIEW

**Comment by huyenltnguyen** [comment]
Hmm, I ran `npm run test` and it looks like there are a lot of failing tests in Portuguese curriculum. Apart from the double quotes escaping issue, the tests also fail due to the [missing challenge description](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/curriculum/challenges/portuguese/08-coding-interview-prep/data-structures/create-and-add-to-sets-in-es6.portuguese.md#description), or missing [localeTitle](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/curriculum/challenges/portuguese/08-coding-interview-prep/rosetta-code/day-of-the-week.portuguese.md), or [invalid test text](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/curriculum/challenges/portuguese/06-information-security-and-quality-assurance/information-security-and-quality-assurance-projects/anonymous-message-board.portuguese.md#tests) or test format (mentioned in #37215).

@RandellDawson I read in #37215 that you're attempting to fix these. Are you in the progress of fixing them? I could give you a hand with this.

**Comment by RandellDawson** [comment]
> @RandellDawson I read in #37215 that you're attempting to fix these. Are you in the progress of fixing them? I could give you a hand with this.

@huyenltnguyen I am almost finished.  I will be submitting a couple of PRs to resolve the various issues found.  I will need to find someone fluent in Portuguese because I had to add several missing `text` sections and several missing `localeTitle` in the frontmatter of challenges.

**Comment by huyenltnguyen** [comment]
@RandellDawson Great! Thanks for taking care of them.

Should I close my PR then, so that it won't cause conflicts with yours? I also attempted to fix some files, but then realized there were just more. I guess I should at least revert them, yeah?
 
> I will need to find someone fluent in Portuguese because I had to add several missing text sections and several missing localeTitle in the frontmatter of challenges.

I always thought we accept Google translated text if the text is served as a placeholder?

**Comment by RandellDawson** [comment]
> I always thought we accept Google translated text if the text is served as a placeholder?

I used Google translate, but I still think we should have someone like @CleoAguiar help us review these before merging.

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/add-rounded-corners-with-border-radius.portuguese.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ localeTitle: Adicionar cantos arredondados com raio de borda
 ```yml
 tests:
   - text: Seu elemento de imagem deve ter a classe "borda verde espessa".
-    testString: 'assert($("img").hasClass("thick-green-border"), "Your image element should have the class "thick-green-border".");'
+    testString: assert($("img").hasClass("thick-green-border"));
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/add-rounded-corners-with-border-radius.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert($("img").hasClass("thick-green-border"), Seu elemento de imagem deve ter a classe "thick-green-border".");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/add-rounded-corners-with-border-radius.portuguese.md [hunk 2]
```diff
@@ -18,9 +18,9 @@ localeTitle: Adicionar cantos arredondados com raio de borda
...
   - text: Sua imagem deve ter um raio de 10 <code>10px</code>
-    testString: 'assert(parseInt($("img").css("border-top-left-radius")) > 8, "Your image should have a border radius of <code>10px</code>");'
+    testString: assert(parseInt($("img").css("border-top-left-radius")) > 8);
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/add-rounded-corners-with-border-radius.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert(parseInt($("img").css("border-top-left-radius")) > 8, "Sua imagem deve ter uma borda com raio de <code>10px</code>");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/size-your-images.portuguese.md [hunk 3]
```diff
@@ -18,9 +18,9 @@ localeTitle: Tamanho suas imagens
 ```yml
 tests:
   - text: Seu elemento <code>img</code> deve ter a classe <code>smaller-image</code> .
-    testString: 'assert($("img[src="https://bit.ly/fcc-relaxing-cat"]").attr("class") === "smaller-image", "Your <code>img</code> element should have the class <code>smaller-image</code>.");'
+    testString: assert($("img[src='https://bit.ly/fcc-relaxing-cat']").attr('class') === "smaller-image");
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/size-your-images.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert($("img[src="https://bit.ly/fcc-relaxing-cat"]").attr("class") === "smaller-image", "Seu elemento <code>img</code> deveria ter a classe <code>smaller-image</code>.");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/size-your-images.portuguese.md [hunk 4]
```diff
@@ -18,9 +18,9 @@ localeTitle: Tamanho suas imagens
...
   - text: Sua imagem deve ter 100 pixels de largura. O zoom do navegador deve estar em 100%.
-    testString: 'assert($("img").width() === 100, "Your image should be 100 pixels wide. Browser zoom should be at 100%.");'
+    testString: assert($("img").width() === 100);
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/size-your-images.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert($("img").width() === 100), "Sua imagem deve ter 100 pixels de largura. O zoom do navegador deve estar em 100%.");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-a-css-class-to-style-an-element.portuguese.md [hunk 5]
```diff
@@ -18,13 +18,13 @@ localeTitle: Use uma classe CSS para estilizar um elemento
 ```yml
 tests:
   - text: Seu elemento <code>h2</code> deve estar vermelho.
-    testString: 'assert($("h2").css("color") === "rgb(255, 0, 0)", "Your <code>h2</code> element should be red.");'
+    testString: assert($("h2").css("color") === "rgb(255, 0, 0)");
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-a-css-class-to-style-an-element.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert($("h2").css("color") === "rgb(255, 0, 0)", "Seu elemento<code>h2</code> deve ser vermelho.");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-a-css-class-to-style-an-element.portuguese.md [hunk 6]
```diff
@@ -18,13 +18,13 @@ localeTitle: Use uma classe CSS para estilizar um elemento
...
   - text: Seu elemento <code>h2</code> deve ter a classe <code>red-text</code> .
-    testString: 'assert($("h2").hasClass("red-text"), "Your <code>h2</code> element should have the class <code>red-text</code>.");'
+    testString: assert($("h2").hasClass("red-text"));
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-a-css-class-to-style-an-element.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert($("h2").hasClass("red-text"), "Seu elemento <code>h2</code> deveria ter a classe <code>red-text</code>.");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-a-css-class-to-style-an-element.portuguese.md [hunk 7]
```diff
@@ -18,13 +18,13 @@ localeTitle: Use uma classe CSS para estilizar um elemento
...
   - text: Sua folha de estilo deve declarar uma classe de <code>red-text</code> e ter sua cor definida como vermelha.
-    testString: 'assert(code.match(/\.red-text\s*\{\s*color\s*:\s*red;\s*\}/g), "Your stylesheet should declare a <code>red-text</code> class and have its color set to red.");'
+    testString: assert(code.match(/\.red-text\s*\{\s*color\s*:\s*red;\s*\}/g));
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-a-css-class-to-style-an-element.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert(code.match(/\.red-text\s*\{\s*color\s*:\s*red;\s*\}/g), "Sua folha de estilo deve declarar uma classe  <code>red-text</code> e tenha a cor definida em vermelho.");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-a-css-class-to-style-an-element.portuguese.md [hunk 8]
```diff
@@ -18,13 +18,13 @@ localeTitle: Use uma classe CSS para estilizar um elemento
...
   - text: 'Não use declarações de estilo inline como <code>style=&quot;color: red&quot;</code> em seu elemento <code>h2</code> .'
-    testString: 'assert($("h2").attr("style") === undefined, "Do not use inline style declarations like <code>style="color&#58; red"</code> in your <code>h2</code> element.");'
+    testString: assert($("h2").attr("style") === undefined);
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-a-css-class-to-style-an-element.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert($("h2").attr("style") === undefined, "Não use declarações de estilo embutido como <code>style="color&#58; red"</code> em seu elemento <code>h2</code>.");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.portuguese.md [hunk 9]
```diff
@@ -18,11 +18,11 @@ localeTitle: Use Seletores de Atributo para Elementos de Estilo
 ```yml
 tests:
   - text: O seletor de atributo de <code>type</code> deve ser usado para selecionar as caixas de seleção.
-    testString: 'assert(code.match(/<style>[\s\S]*?\[type=("|")checkbox\1\]\s*?{[\s\S]*?}[\s\S]*?<\/style>/gi),"The <code>type</code> attribute selector should be used to select the checkboxes.");'
+    testString: assert(code.match(/<style>[\s\S]*?\[type=("|")checkbox\1\]\s*?{[\s\S]*?}[\s\S]*?<\/style>/gi));
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert(code.match(/<style>[\s\S]*?\[type=("|")checkbox\1\]\s*?{[\s\S]*?}[\s\S]*?<\/style>/gi)," O seletor de atributo <code>type</code> deve ser usado para selecionar as caixas de seleção.");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.portuguese.md [hunk 10]
```diff
@@ -18,11 +18,11 @@ localeTitle: Use Seletores de Atributo para Elementos de Estilo
...
   - text: As margens superiores das caixas de seleção devem ser 10px.
-    testString: 'assert((function() {var count=0; $("[type="checkbox"]").each(function() { if($(this).css("marginTop") === "10px") {count++;}});return (count===3)}()),"The top margins of the checkboxes should be 10px.");'
+    testString: assert((function() {var count=0; $("[type=\"checkbox\"]").each(function() { if($(this).css("marginTop") === "10px") {count++;}});return (count===3)}()));
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert((function() {var count=0; $("[type="checkbox"]").each(function() { if($(this).css("marginTop") === "10px") {count++;}});return (count===3)}()),"As margens superiores das caixas de seleção devem ser 10px.");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.portuguese.md [hunk 11]
```diff
@@ -18,11 +18,11 @@ localeTitle: Use Seletores de Atributo para Elementos de Estilo
...
   - text: As margens inferiores das caixas de seleção devem ser de 15px.
-    testString: 'assert((function() {var count=0; $("[type="checkbox"]").each(function() { if($(this).css("marginBottom") === "15px") {count++;}});return (count===3)}()),"The bottom margins of the checkboxes should be 15px.");'
+    testString: assert((function() {var count=0; $("[type=\"checkbox\"]").each(function() { if($(this).css("marginBottom") === "15px") {count++;}});return (count===3)}()));
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert((function() {var count=0; $("[type="checkbox"]").each(function() { if($(this).css("marginBottom") === "15px") {count++;}});return (count===3)}()),"As margens inferiores das caixas de seleção devem ser 15px.");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md [hunk 12]
```diff
@@ -18,15 +18,15 @@ localeTitle: Use seletores CSS para elementos de estilo
 ```yml
 tests:
   - text: Remova o atributo style do seu elemento <code>h2</code> .
-    testString: 'assert(!$("h2").attr("style"), "Remove the style attribute from your <code>h2</code> element.");'
+    testString: assert(!$("h2").attr("style"));
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert(!$("h2").attr("style"), "Remova o atributo style do seu elemento <code>h2</code>.");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md [hunk 13]
```diff
@@ -18,15 +18,15 @@ localeTitle: Use seletores CSS para elementos de estilo
...
   - text: Crie um elemento de <code>style</code> .
-    testString: 'assert($("style") && $("style").length > 1, "Create a <code>style</code> element.");'
+    testString: assert($("style") && $("style").length >= 1);
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert($("style") && $("style").length > 1, "Crie um elemento <code>style</code>.");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md [hunk 14]
```diff
@@ -18,15 +18,15 @@ localeTitle: Use seletores CSS para elementos de estilo
...
   - text: Seu elemento <code>h2</code> deve ser azul.
-    testString: 'assert($("h2").css("color") === "rgb(0, 0, 255)", "Your <code>h2</code> element should be blue.");'
+    testString: assert($("h2").css("color") === "rgb(0, 0, 255)");
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert($("h2").css("color") === "rgb(0, 0, 255)", "Seu elemento <code>h2</code> deveria ser azul.");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md [hunk 15]
```diff
@@ -18,15 +18,15 @@ localeTitle: Use seletores CSS para elementos de estilo
...
   - text: Certifique-se de que a declaração <code>h2</code> sua folha de estilo seja válida com um ponto-e-vírgula e uma chave de fechamento.
-    testString: 'assert(code.match(/h2\s*\{\s*color\s*:.*;\s*\}/g), "Ensure that your stylesheet <code>h2</code> declaration is valid with a semicolon and closing brace.");'
+    testString: assert(code.match(/h2\s*\{\s*color\s*:.*;\s*\}/g));
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert(code.match(/h2\s*\{\s*color\s*:.*;\s*\}/g), "Certifique-se de que sua declaração <code> h2 </code> da folha de estilo seja válida com um ponto-e-vírgula e uma chave de fechamento.");'
```

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md [hunk 16]
```diff
@@ -18,15 +18,15 @@ localeTitle: Use seletores CSS para elementos de estilo
...
   - text: Certifique-se de que todos os seus elementos de <code>style</code> sejam válidos e tenham uma tag de fechamento.
-    testString: 'assert(code.match(/<\/style>/g) && code.match(/<\/style>/g).length === (code.match(/<style((\s)*((type|media|scoped|title|disabled)="[^"]*")?(\s)*)*>/g) || []).length, "Make sure all your <code>style</code> elements are valid and have a closing tag.");'
+    testString: assert(code.match(/<\/style>/g) && code.match(/<\/style>/g).length === (code.match(/<style((\s)*((type|media|scoped|title|disabled)="[^"]*")?(\s)*)*>/g) || []).length);
```
Comment (CleoAguiar) [inline curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md]: I suggest the text translated
```suggestion
    testString: 'assert(code.match(/<\/style>/g) && code.match(/<\/style>/g).length === (code.match(/<style((\s)*((type|media|scoped|title|disabled)="[^"]*")?(\s)*)*>/g) || []).length, "Verifique se todos os seus elementos <code> style </code> são válidos e têm uma tag de fechamento.");'
```

**Review by CleoAguiar — CHANGES_REQUESTED** [review]
I suggest use the translated texts

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-a-css-class-to-style-an-element.portuguese.md (hunk 8 above)
Comment (RandellDawson) [reply curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-a-css-class-to-style-an-element.portuguese.md]: We don't need to add assert messages.  Instead, the `text` section is the assert message.

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.portuguese.md (hunk 9 above)
Comment (RandellDawson) [reply curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.portuguese.md]: We don't need to add assert messages.  Instead, the `text` section is the assert message on the preceding line should be adjusted as needed.

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.portuguese.md (hunk 10 above)
Comment (RandellDawson) [reply curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.portuguese.md]: We don't need to add assert messages.  Instead, the `text` section is the assert message on the preceding line should be adjusted as needed.

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.portuguese.md (hunk 11 above)
Comment (RandellDawson) [reply curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.portuguese.md]: We don't need to add assert messages.  Instead, the `text` section is the assert message on the preceding line should be adjusted as needed.

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md (hunk 12 above)
Comment (RandellDawson) [reply curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md]: We don't need to add assert messages.  Instead, the `text` section is the assert message on the preceding line should be adjusted as needed.

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md (hunk 13 above)
Comment (RandellDawson) [reply curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md]: We don't need to add assert messages.  Instead, the `text` section is the assert message on the preceding line should be adjusted as needed.

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md (hunk 14 above)
Comment (RandellDawson) [reply curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md]: We don't need to add assert messages.  Instead, the `text` section is the assert message on the preceding line should be adjusted as needed.

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md (hunk 15 above)
Comment (RandellDawson) [reply curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md]: We don't need to add assert messages.  Instead, the `text` section is the assert message on the preceding line should be adjusted as needed.

File: curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md (hunk 16 above)
Comment (RandellDawson) [reply curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md]: We don't need to add assert messages.  Instead, the `text` section is the assert message on the preceding line should be adjusted as needed.

**Pull request merged**